aws/rds-postgres/db.tf

```diff
@@ -0,0 +1,516 @@
+    key            = "${local.remote_state_acct_key}/terraform.tfstate"
+    dynamodb_table = "${var.namespace}-master-prd-tf-state-lock"
+    role_arn       = "arn:aws:iam::${var.master_account_id}:role/grv_deploy_svc"
+  }
+}
+
+data "terraform_remote_state" "vpc" {
+  backend = "s3"
+
+  config {
+    region         = "${var.aws_region}"
+    bucket         = "${var.namespace}-master-prd-tf-state-${var.master_account_id}"
+    encrypt        = true
+    key            = "${local.remote_state_vpc_key}/terraform.tfstate"
+    dynamodb_table = "${var.namespace}-master-prd-tf-state-lock"
+    role_arn       = "arn:aws:iam::${var.master_account_id}:role/grv_deploy_svc"
+  }
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# MODULES / RESOURCES
+# ----------------------------------------------------------------------------------------------------------------------
+data "aws_kms_key" "parameter_store_key" {
+  key_id = "alias/parameter_store_key"
+}
+
+module "rds_ssm_param_secret" {
+  source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=0.1.5"
+  kms_arn        = "alias/parameter_store_key"
+  parameter_read = ["/${local.stage_prefix}/rds-secret"]
+}
+
+module "rds_ssm_param_username" {
+  source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=0.1.5"
+  parameter_read = ["/${local.stage_prefix}/rds-username"]
+}
+
+data "aws_iam_policy_document" "rds_ds_access" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["rds.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "rds_ds_access" {
+  count              = 0
+  name               = "rds-ds-access-role"
+  assume_role_policy = "${data.aws_iam_policy_document.rds_ds_access.json}"
+}
+
+resource "aws_iam_role_policy_attachment" "rds_ds_access" {
+  count      = 0
+  role       = "${aws_iam_role.rds_ds_access.name}"
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonRDSDirectoryServiceAccess"
+}
+
+resource "aws_security_group" "this" {
+  count       = "${local.enable_create_security_group}"
+  name        = "rds-security-group"
+  description = "Allow internal and VPN traffic"
+  vpc_id      = "${coalesce(var.vpc_id, data.terraform_remote_state.vpc.vpc_id)}"
+
+  ingress {
+    from_port   = "${var.port}"
+    to_port     = "${var.port}"
+    protocol    = "6"
+    cidr_blocks = ["${var.ingress_sg_cidr}"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["${var.ingress_sg_cidr}"]
+  }
+}
+
+module "db_subnet_group" {
+  source       = "./modules/db-subnet-group"
+  create       = "${local.enable_create_db_subnet_group}"
+  stage_prefix = "${local.stage_prefix}"
+  subnet_ids   = ["${data.terraform_remote_state.vpc.vpc_private_subnets}"]
+  tags         = "${local.tags}"
+}
+
+module "db_parameter_group" {
```

Comment:
I thought Postgres doesn't support parameter groups, is this still needed?

Reply:
Postgres does support parameter groups. It does not support option groups.